### ISSUE TITLE
The Luminosity ratio of Table data text to its background color is less than required ratio i.e; 4.267:1: A11y_WinForm.NETRuntime_Contrast(Minimum)

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#14045]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG1.4.3; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Contrast(Minimum); #A11yMAS;

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle. 
2. Splash screen appears and SSMA is launched 
3. Navigate to connect to oracle and connect to target button and connect it. 
4. Navigate to 'Tools' menu and select it. 
5. Navigate to 'Convert schema' menu item and select it. 
6. Navigate all controls after selecting 'Convert schema'. 
7. Verify whether luminosity ratio is sufficient or not for table data text with its background color. 

### Actual Result:
The Luminosity ratio of Table data text to its background color is less than required ratio i.e.; 4.267:1 

### Expected Result:
The Luminosity ratio of table data text to its background color should be at least 4.5:1 or more.

### User Empact:
A luminosity ratio of 4.267:1 for table data text falls below the required minimum contrast ratio of 4.5:1 for normal text, making it difficult for users with low vision or color blindness to read the content. This can hinder information access and fails to meet WCAG 2.1 AA contrast requirements.

### Sample App used:-

[[[[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=M4uY2O)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=cCZLff)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=wBZc6R)

### Attachment:

<img width="800" height="497" alt="Image" src="https://github.com/user-attachments/assets/327ff4fe-103e-447b-b8f0-6f5ba4c3b7bb" />

**Comment by PoojaNamde** [comment]
Logged this bug to track :https://github.com/dotnet/winforms/issues/13759

**Comment by LeafShi1** [comment]
Starting with .NET 8.0, this contrast has been sufficient.

<img width="2035" height="1272" alt="Image" src="https://github.com/user-attachments/assets/29d3ed78-0288-4252-801d-0f68c9975bf9" />

**Comment by PoojaNamde** [comment]
#A11yFixed; #Closed; #Regressed:11-17-25;

Verified the issue in the latest app. Issue is fixed hence closing the bug.

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/05f69fe5-e064-4284-b66f-4b5775408b28" />
